bin/cron/with_pg/dms_tasks

```diff
@@ -9,9 +9,12 @@ require File.expand_path('../../../../deployment', __FILE__)
 require 'cdo/chat_client'
 require 'cdo/only_one'
 require 'cdo/rake_utils'
-require 'cdo/redshift'
 require 'cdo/slack'
 
+RakeUtils.with_bundle_dir(File.dirname(__FILE__)) do
+  require 'cdo/redshift'
+end
```

Comment:
Interesting, i'm actually surprised it worked like this! I'll keep this pattern in mind for the future.